packages/next/src/server/app-render/app-render.tsx

```diff
@@ -151,24 +153,21 @@ function hasLoadingComponentInTree(tree: LoaderTree): boolean {
   ) as boolean
 }
 
-export type AppPageRender = (
+type AppRenderContext = {
+  staticGeneration: StaticGenerationStore
```

Comment:
nit: I feel like we fairly consistently refer to this as `staticGenerationStore` when referencing it, so not sure it's worth only renaming in `app-render`. Not strongly opinionated here though, just feels like `staticGenerationStore.isStaticGeneration` reads better than `staticGeneration.isStaticGeneration`

Reply:
This was cherry-picked from a set of larger changes I wanted to make that updated the context handling, I've reverted the name change here now :)